addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -1277,7 +1277,16 @@
                         </em>
                         <t t-esc="line.get_unit().name" />
                         at
-                        <t t-esc="widget.format_currency(line.get_unit_display_price(),'Product Price')" />
+                        <t t-if="line.display_discount_policy() == 'without_discount' &amp;&amp; 
+                            line.get_unit_display_price() != line.get_list_price()">
+                            <strike>
+                                <t t-esc="widget.format_currency(line.get_list_price(),'Product List Price')" />
```

Comment:
The second argument to `format_currency` should be the name of a decimal precision. `Product List Price` does not exist, I think it should still be `Product Price` like in the the other calls.